app/src/main/java/com/github/featuredetectandroid/ui/AppLayout.kt

```diff
@@ -63,8 +70,21 @@ fun AppLayout(
                 Image(
                     bitmap = bitmap,
                     contentDescription = "Grayscale photo",
-                    modifier = Modifier.fillMaxSize()
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f)
                 )
+
+                if (keypointOffsets.isNotEmpty()) {
```

Comment:
Ok, it's probably better to leave this as it is because even if the algo has changed the keypoints may still be present (we set them to `emptyList` when this happens but still)